### ISSUE TITLE
Plans for Apple TV 4?

### ISSUE THREAD

**Issue by JakeRound** [OstlerDev/PopcornTV#227]
Might be a long shot but the new Apple TV  4has it's own SDK any plans on development for an app?

Obviously it won't be allowed on the App Store but users with a mac can actually load the Application onto the device via the SDK without requiring a developers account?

**Comment by OstlerDev** [comment]
I plan on starting to port it, however I need to look into how it is going to work first. In order to really do this I would need the new Apple TV. Because I do not have the best financial standing right now, I cannot afford to spend the money on the new Apple TV (and possibly the developers license if needed). As such I have created a new goal on the monthly campaign in order to get funding for the new Apple TV :)

If you can help donate then I can start on it sooner :) https://salt.bountysource.com/teams/popcorntv 

(I am posting this here to respond to you, but it is also a general response to all others that have asked)

**Comment by JakeRound** [comment]
How's the progress on the native application, i saw on the blog it was abandoned due to the lack of an API since they were taken down. But now an API has been rebuilt what's the current state? Also any plans to start sharing on Github?

**Comment by OstlerDev** [comment]
After the unfortunate events of what happened to the guys over at popcorntime.io I decided that it would be safest to build the new version for [Butter]() and to release it that way. I have not stopped work on it, and have been also working on the Butter for iOS app, which will go hand in hand with the tvOS version.

If you have any more questions feel free to ask them :)

**Comment by JakeRound** [comment]
I saw that you were working on the iOS application, good job so far! So from what i understand, Butter it's going to be a completely legal application which will support 3rd party 'plugins' to connect to movie & tv api's? So in the future we'll hopefully have Butter working on iOS/TVos?

Thanks

**Comment by OstlerDev** [comment]
Think of Butter as all the same technology as Popcorn Time, but without any sources that link to illicit material. Butter will never add illicit content, and as such most people do not see it as that desirable. However, the idea is that anybody can come along and add their own sources for Butter and redistribute. They could add Youtube as a source, or someone could add kat.cr as a source and distribute.

Basically Butter is all the good technology without any risky parts. I have debated stopping maintaining PTV just out of worry for legal risk, I have not 100% decided if I should continue working on pTV, or just move it over to ButterTV and allow others to add "plugins".

I hope that clears the air a little.

**Comment by JakeRound** [comment]
I'd have to agree! I'm assuming we won't be seeing it for another couple of months yet? On the other hand iOS app on is Flawless.
